src/wirecraft/client/ui/cable.py

```diff
@@ -27,26 +33,73 @@ def update_position(self, camera: Camera, resolution: Resolution) -> None:
         updated using world coordinates. If the cable is being placed, only the start
         point should be updated (end point follows mouse).
         """
-        # screen_size = resolution.size  # Get the tuple from Resolution object
 
-        # First time update: calculate and store world coordinates
-        if self.start_world is None:
-            # self.start_world = camera.screen_to_world(self.start, screen_size)  # TODO: use int
-            pass
-        if self.ended and self.end_world is None:
-            # self.end_world = camera.screen_to_world(self.end, screen_size)  # TODO: use int
-            pass
+    # this function looks unnecessary as it works fine without it but to be confirmed
+    # screen_size = resolution.size  # Get the tuple from Resolution object
+    # screen_size = resolution.size  # Get the tuple from Resolution object
 
-        # Update start point screen position
-        # self.start = camera.world_to_screen(self.start_world, screen_size)  # TODO: use int
-
-        # Update end point screen position only if cable is complete
-        if self.ended and self.end_world is not None:
-            # self.end = camera.world_to_screen(self.end_world, screen_size)  # TODO: use int
-            pass
+    #
+    ## First time update: calculate and store world coordinates
+    # if self.start_world is None:
+    #    self.start_world = camera.screen_to_world(self.start, screen_size)
+    # if self.ended and self.end_world is None:
+    #    self.end_world = camera.screen_to_world(self.end, screen_size)
+    #
+    ## Update start point screen position
+    # self.start = camera.world_to_screen(self.start_world, screen_size)
+    #
+    ## Update end point screen position only if cable is complete
+    # if self.ended and self.end_world is not None:
+    #    self.end = camera.world_to_screen(self.end_world, screen_size)
```

Comment:
is this code deprecated ?

Reply:
"# this function looks unnecessary as it works fine without it but to be confirmed"